core/src/text/fontContext.cpp

```diff
@@ -2,8 +2,13 @@
 #define FONTSTASH_IMPLEMENTATION
 #include "fontstash.h"
 
+#define LOGE(fmt, ...) do { logMsg("FontContext/Error: " fmt "\n", ## __VA_ARGS__); } while(0)
+#define LOGW(fmt, ...) do { logMsg("FontContext/Warn: " fmt "\n", ## __VA_ARGS__); } while(0)
```

Comment:
I'll look into cmake for setting compiler specific flags. This one needs 
-Wgnu-zero-variadic-macro-arguments, but gcc will complain about unknown flags..